pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "cognite-extractor-utils"
-version = "3.0.0-beta"
+version = "4.0.0-beta"
```

Comment:
I'd prefer `3.0.0-beta2`, or something along those lines. We're already on a beta of a major version, no need to skip 3.0.0 entirely.

Reply:
Sure, I also wasn't sure about the versioning here. I guess it doesn't affect the users in a way that requires major version change. 🤔

Reply:
Is it better now?